RELEASING.md

```diff
@@ -0,0 +1,19 @@
+- [Overview](#overview)
+- [Versioning](#versioning)
+- [Releasing](#releasing)
+
+## Overview
+
+This document explains the release strategy for artifacts in this organization.
+
+## Versioning
+
+This respository, as other in this organization follows semantic versioning.
+
+- **major**: Breaking changes
+- **minor**: New features
+- **patch**: Bug fixes
+
+## Releasing
+
+The release process includes a [MAINTAINER](MAINTAINERS.md) pushing a tag to this repository.
```

Comment:
lowercase MAINTAINER

I think you want to explain that the tag causes [a workflow to run and make a release](https://github.com/opensearch-project/opensearch-build-libraries/blob/main/.github/workflows/release.yml).

Reply:
Let me fix that in new PR. Thanks!

Reply:
https://github.com/opensearch-project/opensearch-build-libraries/pull/25